google/resource_container_cluster.go

```diff
@@ -2680,11 +2689,13 @@ func flattenClusterAddonsConfig(c *containerBeta.AddonsConfig) []map[string]inte
 	}
 
 	if c.CloudRunConfig != nil {
-		result["cloudrun_config"] = []map[string]interface{}{
-			{
-				"disabled": c.CloudRunConfig.Disabled,
-			},
+		cloudRunConfig := map[string]interface{}{
+			"disabled": c.CloudRunConfig.Disabled,
+		}
+		if c.CloudRunConfig.LoadBalancerType == "LOAD_BALANCER_TYPE_INTERNAL" {
+			cloudRunConfig["load_balancer_type"] = "internal"
```

Comment:
Since we are hardcoding "internal" here, setting the balancer type to "INTERNAL" will result in a permanent diff. We should commit to one, or pass through the full `LOAD_BALANCER_TYPE_INTERNAL` which will allow us to get rid of the if condition.

Reply:
Good catch. Yea, changed it to `LOAD_BALANCER_TYPE_INTERNAL`